catchpoint-mixin/dashboards.libsonnet

```diff
@@ -0,0 +1,114 @@
+local g = import './g.libsonnet';
+local logslib = import 'logs-lib/logs/main.libsonnet';
+{
+  local root = self,
+  new(this):
+    local prefix = this.config.dashboardNamePrefix;
+    local links = this.grafana.links;
+    local tags = this.config.dashboardTags;
+    local uid = g.util.string.slugify(this.config.uid);
+    local vars = this.grafana.variables;
+    local annotations = this.grafana.annotations;
+    local refresh = this.config.dashboardRefresh;
+    local period = this.config.dashboardPeriod;
+    local timezone = this.config.dashboardTimezone;
+    local panels = this.grafana.panels;
+    local stat = g.panel.stat;
+    {
+      overview:
+        g.dashboard.new(prefix + ' overview')
+        + g.dashboard.withPanels(
+          g.util.grid.wrapPanels(
+            [
+              panels.topAvgLoadTimeTestName,
+              panels.topAvgTotalLoadTimeNodeName,
+              panels.topAvgDocumentCompletionTimeTestName,
+              panels.topAvgDocumentCompletionTimeNodeName,
+              g.panel.row.new('Requests'),
+              panels.bottomAvgRequestRatioTestName,
+              panels.bottomAvgRequestSuccessRatioNodeName,
+              g.panel.row.new('Connectivity'),
+              panels.topAvgConnectionSetupTimeTestName,
+              panels.topAvgConnectionSetupTimeNodeName,
+              panels.topAvgContentLoadingTimeTestName,
+              panels.topAvgContentLoadingTimeNodeName,
+              panels.topAvgRedirectsTestName,
+              panels.topAvgRedirectsNodeName,
+              g.panel.row.new('Errors'),
+              panels.topErrorsByTestName,
+              panels.alertsPanel,
```

Comment:
I would move the errors row to the absolute top, as it seems like critical information, especially with the alerts panel